src/factories/CompilerFactory.ts

```diff
@@ -505,6 +506,16 @@ export class CompilerFactory {
         return this.jsDocTagInfoCache.getOrCreate(jsDocTagInfo, () => new JSDocTagInfo(jsDocTagInfo));
     }
 
+    /**
+     * Gets a wrapped signature declaration from a compiler signature declaration.
+     * @param compilerObject - Compiler signature declaration.
+     */
+    getSignatureDeclaration(compilerObject: ts.SignatureDeclaration): SignaturedDeclaration {
```

Comment:
`SignaturedDeclaration` is probably wrong.
Which type is the right one?

Reply:
There's no need for this specific method as this functionality already exists in the more general method `CompilerFactory#getNodeFromCompilerNode`. (See my comment in `Signature.ts`)